lib/utils/widgets/theme_mode_picker.dart

```diff
@@ -121,11 +111,17 @@ class _ThemeModePickerState extends State<ThemeModePicker> with SingleTickerProv
     return SizedBox(
       width: width + 48,
       height: height + 48,
-      child: Stack(
-        children: [
-          for (final mode in stacked)
-            if (button(mode) case final button?) button,
```

Comment:
The fact that `button(mode)` and `button?` used the same word caused a bug on the web platform. Now it's fixed! :)